snooty.toml

```diff
@@ -5,3 +5,4 @@ title = "MongoDB Drivers"
 version = 4.0
 package-name-org = "docs-ecosystem"
 pgp-version = "{+version+}"
+toc_landing_pages = ["/drivers/python"]
```

Comment:
You need to use the path to the file, not the path to the URL, so this should just be `"/python"` and then everything will work.